src/ceph-disk/ceph_disk/main.py

```diff
@@ -4540,7 +4542,7 @@ def list_dev(dev, uuid_map, space_map):
     return info
 
 
-def list_devices():
+def list_devices(cluster):
```

Comment:
ahh, right. we should update the mock function of `list_devices_return()` as well